deployer/templates/es.yaml

```diff
@@ -147,9 +150,14 @@ objects:
     description: "Numeric ID of a supplemental group for storage volume access."
     name: STORAGE_GROUP
     value: ${STORAGE_GROUP_DEFAULT}
+  -
+    description: "The mimimum number of masters that must be present in the ES cluster for the master election process to complete"
```

Comment:
If we can, tying it back to the actual Elasticsearch parameter name it lands in would be really great, too.